common/character_interactions/00_alliance.txt

```diff
@@ -97,6 +97,11 @@
 				}
```

Comment:
https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/352#discussion_r628895781

So I analyzed this problem more. You created it here.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/117644589-af5b2c80-b19a-11eb-92eb-c502c77d5cb6.png)

</details>

Such a thing just doesn't exist in the vanilla. `scheme_discovered` is added only in this place and has only one value, `yes` or doesn't exist.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/117644779-ea5d6000-b19a-11eb-9c91-3b24ffb323b1.png)

</details>

Since you share many events with the vanilla, you may get more problems in the future if you don't replace `scheme_discovered` checking `yes` or `no`.

Also, you shouldn't pass `DISCOVERED` to `stasis_trap_success_effect`.

Reply:
Also, I should notice that the originally stated problem is still present.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/117647146-a91a7f80-b19d-11eb-9f72-239dec7c9032.png)

</details>

Also, notice it's a number now.